_episodes/l1-04-managing_contributions.md

```diff
@@ -0,0 +1,154 @@
+> 2. Select an owner for the forked repository (if you belong to any GitHub organisations, they will appear here) and give it a suitable name.
+> ![Create a fork with repository name emphasised]({{ site.baseurl }}/fig/fork_choose_owner_name.png "Create a fork with repository name emphasised"){:class="img-responsive"}
+> 3. Adding a description for your fork is optional. There is also a checkbox asking if you want to copy only the default branch of the repository (in this instance this is called `master`) or whether you want to copy all the branches. In most cases you will only want to copy the default branch. This option is selected by default. Finally, click **Create fork**.
+> ![Create a fork with description and create button]({{ site.baseurl }}/fig/fork_description_create.png "Create a fork with description and create button"){:class="img-responsive"}
+{: .challenge}
+
+After forking the repository, the second step is to make our fix/changes. First we will need to clone **our fork** so that we have the files in that repository locally on our computer (`clone` command was covered in the [introductory course](https://imperialcollegelondon.github.io/introductory_grad_school_git_course/l2-02-remote_repositories/index.html)). From here we can go ahead and create a new fix/feature branch and make our changes. When we are happy with the changes we have made, we can `commit` and `push` our upstream, forked repository.
+
+The third and final step in the workflow is to create a pull request. This is done in the same way as in the shared repository model above (navigate to your forked repository, click on the "Contribute" drop-down menu, then click the "Open pull request" button), only this time instead of the base branch being one in your repository, it is a branch in the upstream repository that you forked.
+
+![Drop-down menus for choosing the base fork and branch]({{ site.baseurl }}/fig/fork_pr_choose_branch.png "Drop-down menus for choosing the base fork and branch"){:class="img-responsive"}
+
+Another difference with pull requests from forked repositories is that you can allow anyone with push access to the upstream repository to make changes to your pull request. This is done by selecting **Allow edits from maintainers**.
+
+![Allow maintainers to make edits checkbox]({{ site.baseurl }}/fig/maintainer_edits.png "Allow maintainers to make edits checkbox"){:class="img-responsive"}
+
+### Requesting reviewers
+- When opening a PR, you can request it to be reviewed by someone else, so there is another pair of eyes making sure that your contribution is correct and does not introduce any bugs.
+- Reviewers can just comment on the PR, approve it, or request changes before it can be approved.
+- Some repositories might require the approval of one or more reviewers before the changes can be merged into the target branch.
```

Comment:
Just a suggestion.
```suggestion
- Some repositories might require the approval of one or more reviewers before the changes can be merged into the target branch. This can be set up by the repository manager(s) as a [branch protection rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule).
```